elections/2022-SC/README.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Knative Steering Committee Election - 2022 Voters Guide"
+linkTitle: "2022 Voters Guide"
+weight: 30
+type: "docs"
+---
+
+# 2022 Voters Guide - Knative Steering Committee Election
+
+## Purpose
+
+The role of this election is to fill out the two (2) seats due for election
+this year on the [Knative Steering Committee]. Each elected member will serve a
+two (2) year term.  
+
+## Background
+
+This election will shape the future of Knative as a community and project. The
+Knative Steering Committee (SC) is responsible for the general health of the
+Knative community. 
+
+## Eligibility
+
+Please refer to the [SC Election Charter] for:
+
+- [Eligibility for candidacy]
+- [Eligibility for voting]
+
+The list of eligible voters is stored [in the community repository](./voters.yaml), but it is easier
+to check if you are eligible by logging into [Elekto]. See Voting Process below for more details.
+
+## Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| October 20   | Announcement of Election, call for nominations, exceptions |
+| November 8   | All candidate nominations due by 23:59 UTC (3:59pm Pacific) |
+| November 10  | Election Begins via Elekto UI |
+| November 28  | Voter exception requests due by 23:59 UTC (3:59pm Pacific) |
+| November 30  | Election Closes by 23:59 UTC (3:59pm Pacific) |
+| December 6   | Announcement of Results |
+
+## Election Officers
+
+In an effort to run a fair and transparent election, the following people
+have been asked by the Steering Committee to run this election:
+
+- [Josh Berkus](https://github.com/jberkus)
+- [Dawn Foster](https://github.com/geekygirldawn)
+- TBD
```

Comment:
@upodroid and @nader-ziada showed interest to help as officers in Slack

Reply:
Exactly! But, the Steering Committee needs to approve / appoint new Election Officers, and we only need one more election officer, so I'm waiting on the official decision from the SC about which one they have selected :)

Reply:
@geekygirldawn looking at the [STEERING-COMMITTEE.md](https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md#election-officers) doc, it's not clear to me if there needs to be a PR of some sort to appoint the election officer. However, in the Steering Committee Slack channel @upodroid was agreed on by 3 members (myself, @csantanapr and @vaikas). Not sure if that is considered quorum given that there are 6 members at the moment. /cc @pmorie @thisisnotapril @itsmurugappan

Reply:
Yeah, you need one more vote, please.

Reply:
Steering appoints @upodroid as election officer, we just had quorum 
@geekygirldawn and @jberkus please move forward and onboard @upodroid 
Thank you @upodroid for volunteering 👍